### DISCUSSION TITLE
plotly modebar fix?

### DISCUSSION THREAD

**Discussion by abiller** [datapane/datapane#162] [Help] [answered]
Hi, 

I'm currently in the process of evaluating datapane to use for automatic report generation. 
There are a few issues which I will open a new discussion for, in due time, but for starters, a simple question -

I'm wrapping Plotly's plotly.graph_objs.Figure with dp.Plot and everything is peachy, except that the modebar is missing (i.e., the manual plotly controls). 

Do you know of a way to fix this? I do not want to call fig.show() for those objects.

PS, it seems that for Bokeh you do retain the controls.

Lemme know if there is anything I could check out on my own to find out!

**Comment by mands** [comment]
Hi @abiller 

Apologies for the delay in getting back to you - I think this may just be a bug / toggle we can enable in our Plotly wrapper. We'll take a look and see if can improve this.

Cheers!

**Comment by johnstaa** [comment]
Has there been any update to this?

**Reply by Buroni** [reply]
Hey @johnstaa yep the modebar is now visible if you hover over the plotly chart.

**Comment by Buroni** [comment, marked as answer]
Modebar is now visible on hover, for example see https://datapane.com/reports/J358eLA/plotly-report/
